docs/features/dead-letter-queue.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: 'Dead-letter queue'
+sidebar_position: 4
+---
+
+A dead-letter queue is a connector to which Conduit redirects records that can't
+be delivered or processed successfully. Any existing Conduit destination
+connector can be used for the purpose of a dead-letter queue.
+
+For a record to be considered processed successfully it needs to be acked. A
+record is acked (acknowledged) when it reaches all destinations in a pipeline
+or it is filtered out by a processor. Likewise, a record is nacked (negatively
+acknowledged) when a processor or destination return an error while processing
+the record. In other words, an ack indicates success and a nack indicates
+failure.
+
+When a record can't be delivered or processed, it is sent to the dead-letter
+queue connector so that it can be handled separately from the other records in
+the pipeline. This can be useful for debugging, since it allows you to isolate
+records that are causing problems and figure out why they are failing.
+Additionally, it can help to prevent the rest of the records in the pipeline
+from being blocked or delayed due to the failure of a single record.
+
+## Nack window and threshold
+
+Rerouting records to a dead-letter queue causes the pipeline to keep running
+despite of nacked (negatively acknowledged) records. However, experiencing a lot
+of nacked records can indicate that a pipeline is corrupted or malfunctioning.
+In such a case it might be better to stop the pipeline to prevent further errors
+and investigate the problem before resuming.
+
+Conduit can monitor a fixed number of last records and track which ones are
+acked or nacked. As new records are processed, they are added into the window
+while the oldest ones are discarded to make room for them. This allows the
+window to "slide" through records, always containing the latest records. Conduit
+keeps track of the number of nacks in the sliding window and stops the pipeline
+if the threshold gets exceeded. This can be useful for ensuring that the
+pipeline is operating correctly and effectively.
+
+Below is a visual representation a sliding window monitoring 5 records with the
+nack threshold set to 2. Notice that once the threshold is exceeded the pipeline
+stops.
+
+![nack window animation](/images/dlq-window-animation.gif)
+
+## Configuring a dead-letter queue
+
+By default a Conduit pipeline doesn't have a dead-letter queue. You can enable
```

Comment:
> By default a Conduit pipeline doesn't have a dead-letter queue.

Nitpick: explicitly stating that this means that the first error stops a pipeline would be nice.

Reply:
Good idea!

Reply:
Reads great!